server/config/config.go

```diff
@@ -145,8 +145,6 @@ type Config struct {
 	Dashboard DashboardConfig `toml:"dashboard" json:"dashboard"`
 
 	ReplicationMode ReplicationModeConfig `toml:"replication-mode" json:"replication-mode"`
-	// EnableRedactLog indicates that whether redact log, 0 is disable. 1 is enable.
-	EnableRedactLog bool `toml:"enable-redact-log" json:"enable-redact-log"`
```

Comment:
Is this configuration compatible with the previous one?

Reply:
As `EnableRedactLog` haven't been released yet, I think it is not necessary for us to consider being compatible here.